frontend/src/pages/books/BookAdd.tsx

```diff
@@ -280,6 +281,9 @@ export default function BookAdd() {
     BOOKS_API.addBookInitialLookup({ isbns: textBox })
       .then((response) => {
         setIsLoadingButton(false);
+
+        const _data = [...books];
+        setBooks(_data);
```

Comment:
Why was this added? Don't we already set all of the books in the loop below?

Reply:
This was added to have the funtionality where clicking lookup didn't erase the currently filled in set of data for the edit book fields data. That way the user has the flexibility to delete individual line items from earlier lookups, and add more books to the edit fields table with additional lookup requests.